ioc.go

```diff
@@ -161,7 +161,7 @@ func executeFixedCommand(t string) {
 	case "add configuration":
 		fallthrough
 	case "new configuration":
-		commands.AddClusterConfiguration(api, username)
+		commands.AddClusterConfigurationCommand(api, username)
```

Comment:
As I see, all other commands don't has suffix `Command`. Do we need it, because this is not consistent?

Reply:
TBH dunno. On one hand, it is common in other languages, OTOH it is discouraged in Go community to repeat package name in function name. I used Command suffix there to distinguish it from the implementation of the operation. It might be removed though.

Reply:
So maybe suffix should be in name of other commands too?

Reply:
According to Go language community, it should be better avoided:

"Avoid stutter. Since client code uses the package name as a prefix when referring to the package contents, the names for those contents need not repeat the package name. The HTTP server provided by the http package is called Server, not HTTPServer. Client code refers to this type as http.Server, so there is no ambiguity."

Reply:
To not introduce incostencies, I'll rename `AddClusterConfigurationCommand` back to `AddClusterConfiguration`. WDYT?